packages/react-cosmos-component-playground/src/components/FixtureList/persistence.js

```diff
@@ -0,0 +1,50 @@
+import localForage from 'localforage';
+import flatMapDeep from 'lodash.flatmapdeep';
+
+const TREE_EXPANSION_STATE = '__cosmos__tree-expansion-state';
+
+export function getSavedExpansionState(projectKey) {
+  return localForage.getItem(`${TREE_EXPANSION_STATE}-${projectKey}`);
```

Comment:
If we return `|| {}` here we can always expect an Object and never have to do `savedExpansionState || {}` when consuming the expansion state.